drawBot/__init__.py

```diff
@@ -1,3 +1,125 @@
 from .drawBotDrawingTools import _drawBotDrawingTool
 
 _drawBotDrawingTool._addToNamespace(globals())
+
+
+# This section is automatically generated with scripting/generateDrawbotInit.py
```

Comment:
Would be good to have a test that verifies that the generated module still matches the current module, so we can't forget when it needs to be done.

Reply:
good idea!

Reply:
fixed with f9276587a7d89db9b47387acdcb19deaea2b08f6